src/content/docs/integrations/mlops-integrations/algorithmia-mlops-integration.mdx

```diff
@@ -7,42 +7,42 @@ tags:
+MLOps provides a tool for monitoring and observing the performance and effectiveness of machine-learning models in a production environment. This increases the possibilities for collaboration between data science and DevOps teams, feeding into a continuous process of development, testing, and operational monitoring.
 
-MLOps allows teams to alleviate these challenges by providing a technological backbone for managing the machine learning lifecycle through automation and scalability. This allows for a seamless collaboration between data teams and DevOp teams ultimately leading to innovative solutions through the creation of sophisctated ML models, continued development and testing, and consistent operational and monitoring of the software. 
+## The Algorithmia integration [#algorithmia]
 
+Algorithmia Insights provides tools for deploying your machine-learning models into production. By integrating, Algorithmia's integration with New Relic, you'll be able to instrument, analyze, troubleshoot, and optimize your machine-learning performance across your entire system. By rigorously observing your capabilities, you'll be able to react quickly to changes in the model's input or output and the relationship between the two.
 
-## Integration with Algorithmia
+Send your model performance metrics from Algorithmia Insights to New Relic and you'll have real-time monitoring for your algorithms. You'll explore your metrics data with user-friendly charts and learn the state of your algorithms at a glance for faster and more efficient troubleshooting. 
 
-After deploying ML models into production, tracking model performance is an underestimated challenge for ML practitioners. Without rigorous observability capabilities, organizations risk failure to react to changes in the model input or output, or the model’s ability to capture the relation between these two.  
+## Integrate Algorithmia with New Relic [#integrate-algorithmia] 
 
-Since New Relic’s cloud-based observability infrastructure already helps instrument, analyze, troubleshoot, and optimize your entire software stack, Algorithmia is the perfect solution to collect model performance data and connect with New Relic’s monitoring tools - allowing for a smooth integration for business and operational management. 
-
-By sending your model performance metrics from Algorithmia Insights to New Relic and having real-time monitoring for your algorithms, you can explore your Metrics data with user-friendly charts, and quickly learn the state of your algorithms for faster and more efficient troubleshooting. 
+First, Algorithmia uses a Kafka topic to stream Insights. Your machine-learning algorithm's performance metrics. Then, the New Relic connector (another algorithm) transforms the Kafka topic into a metrics data payload for a specific New Relic account.
```

Comment:
I don't understand this paragraph. What is the second sentence? There is no predicate...

Reply:
Ha! All the signs of a distracted writer :) I fixed this.